src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,188 @@
-<div class="cd-col-form"
-     *cdFormLoading="loading">
-  <form name="userForm"
-        #formDir="ngForm"
-        [formGroup]="userForm"
-        novalidate>
-    <div class="card">
+<div cdsCol
+     [columnNumbers]="{md: 4}">
+  <ng-container *cdFormLoading="loading">
+    <form #frm="ngForm"
+          #formDir="ngForm"
+          [formGroup]="userForm"
+          novalidate>
       <div i18n="form title"
-           class="card-header">{{ action | titlecase }} {{ resource | upperFirst }}</div>
-      <div class="card-body">
-
-        <!-- Username -->
-        <div class="form-group row">
-          <label class="cd-col-form-label"
-                 [ngClass]="{'required': mode !== userFormMode.editing}"
-                 for="username"
-                 i18n>Username</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   type="text"
-                   placeholder="Username..."
-                   id="username"
-                   name="username"
-                   formControlName="username"
-                   autocomplete="off"
-                   autofocus
-                   ngbTooltip="White spaces at the beginning and end will be trimmed"
-                   i18n-ngbTooltip
-                   cdTrim>
```

Comment:
i see this tooltip and `cdTrim` is also being removed. Probably we shouldn't do it since @Pegonzal  added it intentionally to fix a bug while back. That message and the trim should be happening to the form.

Reply:
tooltip and cdTrim both added